invenio_records_files/utils.py

```diff
@@ -40,7 +45,13 @@ def sorted_files_from_bucket(bucket, keys=None):
 
 
 def record_file_factory(pid, record, filename):
-    """Get file from a record."""
+    """Get file from a record.
+
+    :param pid: TODO: Not used.
```

Comment:
We can call it `dummy` or `_` and in docstring say that it's just to keep function signature.